### PR TITLE
updates CONTRIBUTING.md via #2672

### PR DESCRIPTION
**Pull request by kiddjsh**
## 📝 Description

Add a brief description.

## 💣 Is this a breaking change (Yes/No):

- [ ] Yes
- [x] No

## 🚧 How to migrate?

Don't need.

## 📝 Additional Information

lol

### CODE REVIEW

**Comment by kiddjsh** [comment]
I wanted to go ahead and _include_ some snippets of my illa-builder backend install:

Cloned Repository
gitKrackenClonedRepositoryFiles

pnpm && Node.js LTS Installed
reinstallPNPM NODELTS

NodeLTS v && pnpm v
nodeVersion pnpmVersion

Submodule Init && Submodule Update
gitSubModuleInit SubModuleUpdate

Installed ILLA CLI
launchILLACLI

Created Docker Image
createDockerImage

Deployed on Port 80 && Port 9345
deployedOnPort80 Port9345

./illa doctor Reports Minimum Requirements Satisfied
illaDoctorReportsRequirementDeployILLASatisfied

Created Image Displays in Docker Desktop
illaDockerAllInOneImageCreated

ILLA Builder Banner from Logs
illaBuilderBanner

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,81 +11,200 @@ All work on Illa-builder happens directly on GitHub. Both core team members and
...
 
+The cargo tool will download the ILLA CLI with its source dependencies, build and install it into the cargo bin path so that we can run it. 
 
+Once installed, you can run the ILLA CLI utilizing the illa command:
 
-### If you are go developer
+`./illa <COMMAND>`
+
+Illa CLI is a tool for hosting ILLA Builder in your local environment. Once installed, execute command `./illa deploy --self --port <PORT>`, update `<PORT>` to port number. Again, this will deploy illa-builder-backend at port `<PORT>`. 
+If needing to modify your `.env.self`, input:
```
Comment (AruSeito) [inline CONTRIBUTING.md:198]: If you use the CLI, don't need to care about this configuration file. This file only used on develop or build

Comment (kiddjsh) [reply CONTRIBUTING.md:198]: Should this specific part on the env.self file be removed, and then a new PR submitted?

Comment (donqu1xotechina) [reply CONTRIBUTING.md:198]: Hey @kiddjsh 
We will get this PR merged first and our team will keep improving based on your contribution. Thank you for contributing to our Project!!

**Pull request merged**